app/developers/routes.py

```diff
@@ -617,8 +617,7 @@ def collection_tasklist(collection_id: UUID) -> ResponseReturnValue:
     if form.validate_on_submit():
         try:
             collection_helper.submit_collection(interfaces.user.get_current_user())
```

Comment:
Submitting a collection that's already complete/submitted does nothing - are we happy with that or think we want to raise an error? It's safe, as it stands, but do we want to just ignore that and move on or get notified of the weirdness?